ui/tests/acceptance/secrets/backend/kubernetes/credentials-test.js

```diff
@@ -0,0 +1,77 @@
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import kubernetesScenario from 'vault/mirage/scenarios/kubernetes';
+import ENV from 'vault/config/environment';
+import authPage from 'vault/tests/pages/auth';
+import { fillIn, visit, click, currentRouteName } from '@ember/test-helpers';
+
+module('Acceptance | kubernetes | credentials', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.before(function () {
+    ENV['ember-cli-mirage'].handler = 'kubernetes';
+  });
+  hooks.beforeEach(function () {
+    kubernetesScenario(this.server);
+    this.visitRoleCredentials = () => {
+      return visit('/vault/secrets/kubernetes/kubernetes/roles/role-0/credentials');
+    };
+    this.validateRoute = (assert, route, message) => {
+      assert.strictEqual(currentRouteName(), `vault.cluster.secrets.backend.kubernetes.${route}`, message);
+    };
+    return authPage.login();
+  });
+  hooks.after(function () {
+    ENV['ember-cli-mirage'].handler = null;
+  });
+
+  test('it should have correct breadcrumb links in credentials view', async function (assert) {
+    assert.expect(3);
+    await this.visitRoleCredentials();
+    await click('[data-test-crumb="details"] a');
+    this.validateRoute(assert, 'roles.role.details', 'Transitions to role details route on breadcrumb click');
+    await this.visitRoleCredentials();
+    await click('[data-test-crumb="roles"] a');
+    this.validateRoute(assert, 'roles.index', 'Transitions to roles route on breadcrumb click');
+    await this.visitRoleCredentials();
+    await click('[data-test-crumb="overview"] a');
+    this.validateRoute(assert, 'overview', 'Transitions to overview route on breadcrumb click');
+  });
+
+  test('it should transition to role details view on Back click', async function (assert) {
+    assert.expect(1);
+    await this.visitRoleCredentials();
+    await click('[data-test-generate-credentials-back]');
+
+    await this.validateRoute(assert, 'roles.role.details', 'Transitions to role details on Back click');
+  });
+
+  test('it should transition to role details view on Done click', async function (assert) {
+    assert.expect(1);
+    await this.visitRoleCredentials();
+    this.server.post('/kubernetes-test/creds/role-0', () => {
+      assert.ok('POST request made to generate credentials');
+      return {
+        request_id: '58fefc6c-5195-c17a-94f2-8f889f3df57c',
+        lease_id: 'kubernetes/creds/default-role/aWczfcfJ7NKUdiirJrPXIs38',
+        renewable: false,
+        lease_duration: 3600,
+        data: {
+          service_account_name: 'default',
+          service_account_namespace: 'default',
+          service_account_token: 'eyJhbGciOiJSUzI1NiIsImtpZCI6Imlr',
+        },
+      };
+    });
+    await fillIn('[data-test-kubernetes-namespace]', 'kubernetes-test');
+    await click('[data-test-toggle-input]');
+    await click('[data-test-toggle-input="Time-to-Live (TTL)"]');
```

Comment:
Not entirely sure how the form looks in the UI, but if it's toggled above (and so the input is visible), then I believe you don't need to click here, since `fillIn` below should suffice

Reply:
The first click is for the ClusterRoleBinding toggle in the UI! 😊
<img width="1394" alt="Screen Shot 2022-12-16 at 10 28 37 AM" src="https://user-images.githubusercontent.com/30884335/208164930-b4dc45a2-36d9-4b76-9a48-646be230a6d1.png">

Thanks for reviewing this, Claire!!!